code/modules/mob/living/simple_animal/hostile/tree.dm

```diff
@@ -27,13 +27,27 @@
 	emote_taunt = list("growls")
 	taunt_chance = 20
 
-	//Space carp aren't affected by atmos.
-	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	//Space carp aren't affected by atmos. //Space Carps no. But trees Yes

```

Comment:
Just remove this comment, it's copypasta damage and adding more comments doesn't make it more funny